Makefile

```diff
@@ -78,21 +70,18 @@ endif
 run: kube-router ## Runs "kube-router --help".
 	./kube-router --help
 
-container: Dockerfile.$(GOARCH).run kube-router gobgp multiarch-binverify ## Builds a Docker container image.
+container: kube-router gobgp multiarch-binverify ## Builds a Docker container image.
 	@echo Starting kube-router container image build for $(GOARCH) on $(shell go env GOHOSTARCH)
 	@if [ "$(GOARCH)" != "$(shell go env GOHOSTARCH)" ]; then \
 	    echo "Using qemu to build non-native container"; \
 	    $(DOCKER) run --rm --privileged $(QEMU_IMAGE) --reset -p yes; \
 	fi
-	$(DOCKER) build -t "$(REGISTRY_DEV):$(subst /,,$(IMG_TAG))" -f Dockerfile.$(GOARCH).run .
+	$(DOCKER) build -t "$(REGISTRY_DEV):$(subst /,,$(IMG_TAG))" -f Dockerfile --build-arg ARCH="$(GOARCH)/" .
 	@if [ "$(GIT_BRANCH)" = "master" ]; then \
 	    $(DOCKER) tag "$(REGISTRY_DEV):$(IMG_TAG)" "$(REGISTRY_DEV)"; \
 	fi
 	@echo Finished kube-router container image build.
 
-Dockerfile.$(GOARCH).run: Dockerfile Makefile
-	@sed -e "$(DOCKERFILE_SED_EXPR)" Dockerfile > $(@)
-
```

Comment:
I take it that proper handling of multi-arch builds is still something outstanding on this PR?

Reply:
Yes still outstanding, this only removes the creation of arch specific dockerfiles and moves it to a build-time argument